vars/buildRpm.groovy

```diff
@@ -26,6 +26,11 @@
    * config['description']  Description to report for SCM status.
    *                        Default env.STAGE_NAME.
    *
+   * config['flow_name']    Flow name to use for looking up the log URL
+   *                        for reporting to the SCM.
+   *                        Default is to use config['context'] or its
+   *                        default value.
+   *
```

Comment:
Shouldn't this **always** be the `config['context']` value?  I.e. not even a settable parameter.
```suggestion
```

Reply:
I considered that.   I thought it would be better to have it as an optional parameter to give us more flexibility in its use.

It may be that env.STAGE_NAME is not a good default in a Matrix implementation.

I also wanted to make sure that anyone working with either place in the code understood purpose of the flow name as it is a different entity than the stage name.